components/chainhook-types-rs/src/rosetta.rs

```diff
@@ -341,19 +341,24 @@ pub enum OrdinalInscriptionTransferDestination {
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
 pub enum OrdinalInscriptionCurseType {
-    Tag(u8),
-    Batch,
-    P2wsh,
+    DuplicateField,
+    IncompleteField,
+    NotAtOffsetZero,
+    NotInFirstInput,
+    Pointer,
+    Pushnum,
     Reinscription,
-    Unknown,
+    Stutter,
+    UnrecognizedEvenField,
+    Generic,
 }
```

Comment:
@lgalabru do you have any info on why these changes are necessary?

Reply:
Yeah, we're just aligning our list of curses with the one from `ord`.